### ISSUE TITLE
Bug:  SetMinResize error

### ISSUE THREAD

**Issue by nancikennedy** [BloodDragon2580/Portable#6]
On Retail, with v6.7 installed....

1x Interface/AddOns/Portable/Main.lua:240: attempt to call method 'SetMinResize' (a nil value)
[string "@Interface/AddOns/Portable/Main.lua"]:240: in function `CreateUI_Frame'
[string "@Interface/AddOns/Portable/Main.lua"]:221: in function `CreateUI'
[string "@Interface/AddOns/Portable/Main.lua"]:180: in function `?'
[string "@Interface/AddOns/Portable/Main.lua"]:155: in function <Interface/AddOns/Portable/Main.lua:153>

Locals:
self = <table> {
 Slash_Help = <function> defined @Interface/AddOns/Portable/Main.lua:1362
 Do_ReorderList = <function> defined @Interface/AddOns/Portable/Reorder.lua:176
 UpdateUI_ButtonGrid = <function> defined @Interface/AddOns/Portable/Main.lua:709
 print = <function> defined @Interface/AddOns/Portable/Common.lua:7
 UpdateUI_Broker = <function> defined @Interface/AddOns/Portable/Broker.lua:111
 SetFrameStyle = <function> defined @Interface/AddOns/Portable/Common.lua:82
 ui = PortableUIFrame {
 }
 DoScript_OnEnter = <function> defined @Interface/AddOns/Portable/Main.lua:1274
 CreateUI_Container = <function> defined @Interface/AddOns/Portable/Main.lua:356
 CreateUI_Reorder = <function> defined @Interface/AddOns/Portable/Reorder.lua:59
 MAX_BUTTONS = 15
 CreateUI_Sizer = <function> defined @Interface/AddOns/Portable/Main.lua:316
 Options = <table> {
 }
 DoScript_OnClick = <function> defined @Interface/AddOns/Portable/Main.lua:1294
 UpdateUI_MinimapIcon = <function> defined @Interface/AddOns/Portable/MinimapIcon.lua:41
 HideUI_Broker = <function> defined @Interface/AddOns/Portable/Broker.lua:338
 UpdateUI_ShowTextures = <function> defined @Interface/AddOns/Portable/Main.lua:548
 UpdateUI = <function> defined @Interface/AddOns/Portable/Main.lua:417
 CreateUI_MinimapIcon = <function> defined @Interface/AddOns/Portable/MinimapIcon.lua:28
 GetAddonInfo = <function> defined @Interface/AddOns/Portable/Common.lua:24
 MakeTooltip = <function> defined @Interface/AddOns/Portable/Common.lua:63
 Helper_ResetLearning = <function> defined @Interface/AddOns/Portable/Main.lua:1151
 LDB = <table> {
 }
 Helper_EscapeToClose = <function> defined @Interface/AddOns/Portable/Main.lua:1238
 Helper_ShowConfig = <function> defined @Interface/AddOns/Portable/Main.lua:1230
 Helper_GetToolbarSize = <function> defined @Interface/AddOns/Portable/Main.lua:1215
 Helper_IsInBags = <function> defined @Interface/AddOns/Portable/Main.lua:1201
 UpdateUI_TextStyle = <function> defined @Interface/AddOns/Portable/Main.lua:1033
 Helper_UpdateBrokerInfoText = <function> defined @Interface/AddOns/Portable/Broker.lua:306
 Helper_IsSpecialFrame = <function> defined @Interface/AddOns/Portable/Main.lua:1253
 Helper_LearnOrder = <function> defined @Interface/AddOns/Portable/Main.lua:1085
 UpdateUI_FrameSize = <function> defined @Interface/AddOns/Portable/Main.lua:426
 Helper_ShowDisabled = <function> defined @Interface/AddOns/Portable/Main.lua:1058
 MakeText = <function> defined @Interface/AddOns/Portable/Common.lua:53
 UpdateUI_ButtonActions = <function> defined @Interface/AddOns/Portable/Main.lua:901
 UpdateUI_FrameStyle = <function> defined @Interface/AddOns/Portable/Main.lua:573
 Apply_Reorder = <function> defined @Interface/AddOns/Portable/Reorder.lua:41
 UpdateUI_IconSizeFrame = <function> defined @Interface/AddOns/Portable/Main.lua:491
 Helper_AdjustFont = <function> defined @Interface/AddOns/Portable/Main.lua:1073
 hSpell = <table> {
 }
 db = <table> {
 }
 L = <table> {
 }
 ShowUI_Broker = <function> defined @Interface/AddOns/Portable/Broker.lua:270
 brokerFeed = <table> {
 }
 DoScript_OnLeave = <function> defined @Interface/AddOns/Portable/Main.lua:1284
 CreateUI_Frame = <function> defined @Interface/AddOns/Portable/Main.lua:232
 Helper_ResetOrder = <function> defined @Interface/AddOns/Portable/Main.lua:1176
 SetFontSize = <function> defined @Interface/AddOns/Portable/Common.lua:102
 ShowReorderUI = <function> defined @Interface/AddOns/Portable/Reorder.lua:13
 CreateUI = <function> defined @Interface/AddOns/Portable/Main.lua:220
 UpdateUI_Container = <function> defined @Interface/AddOns/Portable/Main.lua:539
 aSpell = <table> {
 }
 CreateUI_Broker = <function> defined @Interface/AddOns/Portable/Broker.lua:46
 UpdateOptions = <function> defined @Interface/AddOns/Portable/Main.lua:207
 EventFrame = PortableEventFrame {
 }
 Helper_BrokerWatchMouse = <function> defined @Interface/AddOns/Portable/Broker.lua:319
 CreateUI_Buttons = <function> defined @Interface/AddOns/Portable/Main.lua:362
 lsm = <table> {
 }
 MakeButton = <function> defined @Interface/AddOns/Port

**Comment by BloodDragon2580** [comment]
Sorry cant fix anything at this time. I have no play time for testing.

**Comment by nancikennedy** [comment]
SetMinResize has been superseded by SetResizeBounds.

I replaced the one instance in Main.lua, and reloaded. This error was gone, and I could open up the Portable window. However, it's now having an issue with InterfaceAddOnsList_Update() @ line 1231. I commented that line out, so if I forget and click the options button it will just not open to category the first time. I was also able to reorder things, so even if you don't have time to fix it for the foreseeable future I can still use it. :)
